frontend/src/styles/common.scss

```diff
@@ -15,4 +15,12 @@ $primary: #f5a200;
     min-width: 140px;
     border-radius: 5px;
     font-family: $font-family-sans-serif;
+}
+
+.yellow-hr {
+    margin: 0 !important;

```

Comment:
Why is the !important necessary here? Usually it should be avoided. Otherwise looks good

Reply:
I tried removing all of those !important statements and didn't see any changes so maybe those are unnecessary